src/main/kotlin/com/thelastpickle/tlpstress/commands/Run.kt

```diff
@@ -148,12 +148,15 @@ class Run(val command: String) : IStressCommand {
 
     @Parameter(names = ["--deleterate", "--deletes"], description = "Deletion Rate, 0-1.  Workloads may have their own defaults.  Default is dependent on workload.")
     var deleteRate : Double? = null
-    
+
     val log = logger()
 
     @Parameter(names = ["--max-requests"], description = "Sets the max requests per connection")
     var maxRequestsPerConnection : Int = 32768
 
+    @Parameter(names = ["--connections"], description = "Sets the number of connections per host")
```

Comment:
I really like this option. Thank you for adding it in.

The `setConnectionsPerHost` allows for a core and maximum number of connections to be defined respectively. I think we should still be able to specify both values independently, rather than always set both with a single value.

Would it be possible to allow a singular as well as comma separated value to be passed to this option? In the case where a single value is passed, it would apply to both the core and maximum number of connections. In the case where two comma separated values are supplied, the first value is used for the core number of connections and the second value is used for the maximum number of connections. e.g.

```
# core and maximum number of connections are set to 5
$ tlp-stress run --connections 5 ...

# core number of connections set to 3, maximum number of connections set to 6
$ tlp-stress run --connections 3,6 ...
```

Alternatively, we could have two separate options one for core number of connections and one for maximum number of connections. e.g.

```
$ tlp-stress run --core-connections 3 --max-connections 6 .
```

The second method is simple to implement, easier maintainable, more intuitive than the first.

In addition, I think we should still keep the original values as defaults i.e. the default value for core number of connections be set to 4. The default value for maximum number of connections be set to 8.